recipes/r-harmony/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set version = '1.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-harmony
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/harmony_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/harmony/harmony_{{ version }}.tar.gz
+  sha256: a63c7d7cbbc5d183e8f919552e9d73044e0a89660856e80861a00eb5d25ac7b5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

Comment:
Windows will need:

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'            # [win]
    - '*/Rblas.dll'        # [win]
    - '*/Rlapack.dll'      # [win]

```

Reply:
FYI, here's a reference for this: https://github.com/conda/conda-build/pull/4786